package.json

```diff
@@ -40,8 +40,9 @@
     "@actions/exec": "^1.1.0",
     "@actions/github": "^5.0.0",
     "@actions/io": "^1.1.1",
+    "@oclif/core": "^1.0.11",
     "@octokit/types": "^6.34.0",
-    "bump-cli": "^2.2.6"
+    "bump-cli": "github:paulrbr/cli#public-diff"
```

Comment:
Just to be sure, 
this will be changed after https://github.com/bump-sh/cli/pull/192 is merged and released, right ?

Reply:
yes